src/SprykerEco/Zed/Payone/Business/Api/Response/Container/KlarnaGenericPaymentResponseContainer.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Api\Response\Container;
+
+class KlarnaGenericPaymentResponseContainer extends AbstractResponseContainer
+{
+    public const ADD_PAYDATA_REPLACEMENT_PATTERN = '/add_paydata\[(.*)\]/';
```

Comment:
* If it's not used outside the class then mark it as `protected.

Reply:
++